packages/client/utils/parseEmailAddressList.ts

```diff
@@ -13,7 +13,7 @@ const parseEmailAddressList = (rawStr = ''): {parsedInvitees: any; invalidEmailE
     .replace(/,$/g, '')
 
   const emailCount = commaDelimStr.split(',').length
-  const lastEmailIndex = commaDelimStr.lastIndexOf(' ' || ',')
+  const lastEmailIndex = Math.max(commaDelimStr.lastIndexOf(' '), commaDelimStr.lastIndexOf(','))
```

Comment:
are you sure this works? if both are -1 then it will slice from -1, which is the last char of the string

Reply:
good point! I've updated the Math.max to make sure that we slice from the beginning in that case